core/new-gui/src/app/home/service/google-login.service.ts

```diff
@@ -4,26 +4,19 @@ import { HttpClient } from "@angular/common/http";
 import { AppSettings } from "../../common/app-setting";
 declare var window: any;
 
-export interface credentialRes {
-  client_id: string;
-  credential: string;
-  select_by: string;
-}
 @Injectable({
   providedIn: "root",
 })
-export class GoogleService {
-  private _googleCredentialResponse = new Subject<credentialRes>();
-
+export class GoogleLoginService {
+  private _googleCredentialResponse = new Subject<any>();
   constructor(private http: HttpClient) {}
-
   public googleInit(parent: HTMLElement | null) {
     this.http.get(`${AppSettings.getApiEndpoint()}/auth/google/clientid`, { responseType: "text" }).subscribe({
       next: response => {
         window.onGoogleLibraryLoad = () => {
           window.google.accounts.id.initialize({
             client_id: response,
-            callback: (auth: credentialRes) => {
+            callback: (auth: any) => {
```

Comment:
Using `any` for type is not a good way and should not be recommended.

Reply:
Changed back.